thirdparty/build-thirdparty.sh

```diff
@@ -987,6 +987,63 @@ build_streamvbyte() {
     make install
 }
 
+# jasson
+build_jasson() {
+    check_if_source_exist $JANSSON_SOURCE
+    cd $TP_SOURCE_DIR/$JANSSON_SOURCE/
+    mkdir -p build
+    cd build
+    $CMAKE_CMD .. -DCMAKE_INSTALL_PREFIX=${TP_INSTALL_DIR}
+    ${BUILD_SYSTEM} -j$PARALLEL
+    ${BUILD_SYSTEM} install
+}
+
+# avro-cpp
+build_avro_cpp() {
+    check_if_source_exist $AVRO_SOURCE
+    cd $TP_SOURCE_DIR/$AVRO_SOURCE/lang/c++
+    mkdir -p build
+    cd build
+    $CMAKE_CMD -G "Unix Makefiles" ..
```

Comment:
Why there is no `-DCMAKE_INSTALL_PREFIX=${TP_INSTALL_DIR} -DCMAKE_BUILD_TYPE=Release` options like avro_c

Reply:
I reviewed the dependencies. We don't need avro  cpp interface now, so I have removed the Avro-cpp compilation entry.